src/frontend/tests/playwright/notification_helpers.py

```
@@ -1,21 +1,32 @@
     settle_timeout_ms: int = 100,
+    appear_timeout_ms: int = 1000,
     modal_timeout_ms: int = 5000,
 ) -> None:
     if page.is_closed():
         return
 
     alerts = page.locator("#notification-bar [role='alert']")
+    try:
+        alerts.first.wait_for(state="visible", timeout=appear_timeout_ms)
+    except Exception:
+        pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)